lib/brexit_checker/action.rb

```diff
@@ -29,4 +29,14 @@ def self.load_all
     @load_all = nil if Rails.env.development?
     @load_all ||= YAML.load_file(CONFIG_PATH)["actions"].map { |a| new(a) }
   end
+
+  def add_email_tracking_to_url(non_tracked_url, change_note_id)
+    url = Addressable::URI.parse(non_tracked_url)
+    url.query_values = (callback.query_values || {}).merge({
```

Comment:
The `callback` is mysterious here - perhaps it's a feature of this particular library. To avoid confusion, can we do `url.query_values ||= {}` and then set the other params separately?

Reply:
sorry callback was a test variable - I'll update now.